outlines/blaze.md

```diff
@@ -0,0 +1,56 @@
+# Blaze
+
+1. Introduction to Blaze -- the tracker-backed handlebars like templating syntax
+  1. Example of spacebars syntax, data context + helpers
+  2. Data contexts and access - name lookup (`.`, `this`, `..`, `[0]`), and null values
+  3. Helpers, arguments, options
+  4. Inclusion of other templates + arguments, passing data contexts
+  5. Helpers inside tags -- returning strings and objects, `checked={{isChecked}}`
+  6. Nested helpers + sub expressions
+  7. Block helpers (you can create them with templates, see 10. below)
+  8. Safestrings and `{{{`
+  9. Builtin block helpers
+    1. `{{#if/unless}}`
+    2. `{{#with}}`
```

Comment:
I would agree here somewhat, because in Blaze Components data context is accessible only through explicit `this.data()`. But yea, you do want to access. At least in my common use case for data contexts (I use data context mostly only for this, other communication between components I do differently), which is instances of documents (in my cases instances of PeerDB models). Then you have not just documents, but also methods on documents. So you can structure code in nice way: you have document properties, methods on the model, and component methods. So some code is more suitable to be together with a model, and some to be with view. But it is pretty normal then to call into model methods from template helpers as well.

In some way, logic should be in template helpers in my opinion. Putting too much of it into templates, just so that you can then pass a value to a template helper so that it does not have to access the data context is probably bad.

So what I am saying is that pure Blaze has some shortcomings but that is what we have. So we should be OK with that and explain how to use that effectively, even if it is not the best approach we would take now.